.gitignore

```diff
@@ -1,5 +1,7 @@
 .vscode/
-wurst/war3map.j
-objectEditingOutput/*
-compiled.j.txt
+/_build
 
+*.w3x
+*.w3m
+
+wurst.dependencies
```

Comment:
Please explain the nature of this change - is this file no longer used?

Reply:
it is system specific, and generated.

Reply:
When is it generated? If it's deleted, will the map still build in vscode?